Backend/Interview.Domain/Rooms/RoomReviews/SERoomReviewState.cs

```diff
@@ -5,7 +5,7 @@ namespace Interview.Domain.Rooms.RoomReviews;
 public class SERoomReviewState : SmartEnum<SERoomReviewState>
 {
     public static readonly SERoomReviewState Open = new("Open", 0, EVRoomReviewState.Open);
```

Comment:
pass only enum instead of int value, it can improve code and reduce same errors. For example see SEQuestionType
```c#
private SEQuestionType(string name, EVQuestionType type)
                   // ↓ this is solution
        : base(name, (int)type)
    {
    }
```

Reply:
Yes of course. I will do it.